xml/FHIR-smart-web-messaging.xml

```diff
@@ -1,19 +1,27 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<specification
-    gitUrl="https://github.com/HL7/smart-web-messaging"
-    url="http://hl7.org/fhir/smart-web-messaging"
-    ciUrl="http://build.fhir.org/ig/HL7/smart-web-messaging"
-    defaultWorkgroup="fhir-i"
-    defaultVersion="1.0"
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:noNamespaceSchemaLocation="../schemas/specification.xsd"
->
-  <version code="1.0"/>
-  <artifactPageExtension value="-definitions"/>
-  <artifactPageExtension value="-examples"/>
-  <artifactPageExtension value="-mappings"/>
-  <page name="(NA)" key="NA"/>
-  <page name="(many)" key="many"/>
-  <page name="(profiles)" key="profiles"/>
-  <page name="smart-web-messaging" key="smart-web-messaging" url="smart-web-messaging"/>
+<specification 
+               gitUrl="https://github.com/HL7/smart-web-messaging"
+               url="http://hl7.org/fhir/smart-web-messaging"
+               ciUrl="http://build.fhir.org/ig/HL7/smart-web-messaging"
+               defaultWorkgroup="fhir-i"
+               defaultVersion="1.0.0"
```

Comment:
Ug.  This version change is causing the spec builder to fail because `1.0` has been deleted.

```
[xslt] ERROR: Version with code 1.0 in specification FHIR-smart-web-messaging has been removed or changed.  Versions should never change.  Keys should also not be removed.  Versions can only be removed if no JIRA tracker references that version.  If this is the case and the version should really be removed, please coordinate with an administrator.
```

@lmckenzi - can I coordinate with you to help this get un-stuck?

Reply:
Instead of trying to fix this, I am creating an issue to request some extra XML schema logic to prevent this from happening again by accident.